### PR TITLE
[WebAPI] Add 3 Google Play purchase manual test cases

### PR DESCRIPTION
**Pull request by yhe39**
Add 3 manual test cases for the Google Play purchase API.
And update the test steps.

Impacted tests(approved): new 3, update 0, delete 0
Unit test platform: Crosswalk Project for Android 18.48.477.13
Unit test result summary: Pass 2, fail 1, block 0

BUG=https://crosswalk-project.org/jira/browse/CTS-1552

### CODE REVIEW

**Comment by yhe39** [comment]
@Honry Please help to review. Thanks!

**Comment by Honry** [comment]
Please add test case in tests.xml/tests.full.xml, and don't combine manual and auto tests in one html.

File: webapi/webapi-iap-xwalk-tests/tests.xml [hunk 1]
```diff
@@ -28,11 +28,26 @@
           <test_script_entry>/iap/Navigator_iap_init_error.html</test_script_entry>
         </description>
       </testcase>       
-    <testcase component="Crosswalk APIs/Experimental/InApp Payment API" execution_type="auto" id="Navigator_iap_queryProductsInfo" priority="P1" purpose="Test Navigator iap queryProductsInfo method" status="approved" type="compliance" subcase="6">
+      <testcase component="Crosswalk APIs/Experimental/InApp Payment API" execution_type="auto" id="Navigator_iap_queryProductsInfo" priority="P1" purpose="Test Navigator iap queryProductsInfo method" status="approved" type="compliance" subcase="6">
         <description>
           <test_script_entry>/iap/Navigator_iap_queryProductsInfo.html</test_script_entry>
         </description>
       </testcase>
+      <testcase component="Crosswalk APIs/Experimental/InApp Payment API" execution_type="manual" id="Navigator_iap_purchase" priority="P1" purpose="Test Navigator iap purchase method" status="approved" type="compliance" subcase="1">
```
Comment (Honry) [inline webapi/webapi-iap-xwalk-tests/tests.xml]: No need "subcase", it is used for test which contains more than 1 subcase.

File: webapi/webapi-iap-xwalk-tests/iap/Navigator_iap_purchase.html [hunk 2]
```diff
@@ -29,62 +29,32 @@
 <meta charset="utf-8">
 <title>WebAPI: IAP</title>
 <link rel="author" title="Intel" href="http://www.intel.com">
-<link rel="author" title="Zhong Qiu" href="mailto:zhongx.qiu@intel.com">
+<link rel="author" title="He Yue" href="mailto:yue.he@intel.com">
 <link rel="help" href="http://www.w3.org/TR/IAP-api/">
 <script src="../resources/testharness.js"></script>
 <script src="../resources/testharnessreport.js"></script>
```
Comment (Honry) [inline webapi/webapi-iap-xwalk-tests/iap/Navigator_iap_purchase.html]: Unused references.

File: webapi/webapi-iap-xwalk-tests/iap/Navigator_iap_purchase.html [hunk 3]
```diff
@@ -29,62 +29,32 @@
...
 <div id="log"></div>
```
Comment (Honry) [inline webapi/webapi-iap-xwalk-tests/iap/Navigator_iap_purchase.html]: This line is used when test case reference testharness.js

**Pull request closed without merging**